### ISSUE TITLE
contributor who are author and editor

### ISSUE THREAD

**Issue by dastewo** [UB-Heidelberg/omp#51]
@asmecher
@kadowa 
@AnjaKonopka 
We have a problem with contributors who are both editor and author. While uploading a text you can either give a contributor the author role or the editor role unless you put them twice in the system. But in that case they'll be also shown twice on the frontend, which is a bit annoying especially if you have included the bio.
How is the best way to proceed? Should we decide which role we give them? Or is it better to put them twice in the system? Or is it possible to create a new role which combines both? Where do the roles come from, they are not the same as users enrolled in the systems have.

Kind regards
Daniela

**Comment by asmecher** [comment]
@danielawolf, where do you need this data to be exposed (i.e. showing that the user account was active as both author and editor)?

**Comment by dastewo** [comment]
@asmecher 
I was indeed a bit unclear, it doesn't concern the user accounts. Let me rephrase it: While uploading a manuscript you can enter and change the catalog data. There you can add the contributors and explain their role (Author, Volume editor, Chapter Author, Translator). Now, sometimes people have two roles, for example they are authors of several chapters but also the volume editors. We are not sure how to handle this. Should we add these people twice (one time as author, the other as volume editor)? Or should we stick to one role, even if the book cover says clearly otherwise? 
Can you explain, where these roles come from?

The screenshot is from the old OMP version, but it's the same with the new one:
![bildschirmfoto vom 2016-03-01 08 50 06](https://cloud.githubusercontent.com/assets/15866252/13420915/8e2bba44-df8a-11e5-8748-4d4ff471ce93.png)

Kind regards
Daniela
